packages/Algernon.package/ALGHelpType.class/README.md

```diff
@@ -0,0 +1 @@
+A help item displays a description of a hotwort
```

Comment:
Please say keyword

Reply:
I have not added any class comments... But I adjusted the wording now in that peticular comment

Reply:
Ah, thanks.